contracts/src/v0.8/ccip/test/helpers/Multicall3.sol

```diff
@@ -0,0 +1,245 @@
+// SPDX-License-Identifier: MIT
+// cribbed from https://github.com/mds1/multicall/commit/ebd8b64457454fc10037b3a3ea858f9c08dad4d3
+// changed solidity version to 0.8.24 from 0.8.12
+// ran forge fmt
+// solhint-disable
+pragma solidity 0.8.24;
+
```

Comment:
Maybe Rename it to `MulticallAggregator`? What's the 3 for exactly?

Reply:
See PR description and comment above these lines, this is a 3rd party contract